app/experimenter/experiments/models/nimbus.py

```diff
@@ -238,6 +238,34 @@ def experiment_url(self):
             "https://{host}".format(host=settings.HOSTNAME), self.get_absolute_url()
         )
 
+    def _get_targeting_versions(self):
+        expressions = []
+
+        version_supported = True
+        if self.application in self.TARGETING_APPLICATION_SUPPORTED_VERSION:
+            supported_version = self.TARGETING_APPLICATION_SUPPORTED_VERSION[
+                self.application
+            ]
+            version_supported = self.firefox_min_version >= supported_version
+
+        if version_supported:
+            if self.firefox_min_version:
+                expressions.append(
+                    "version|versionCompare('{version}') >= 0".format(
```

Comment:
```suggestion
                    "app_version|versionCompare('{version}') >= 0".format(
```

This is annoying, but I think it should be `app_version` from https://github.com/mozilla/application-services/blob/main/components/nimbus/src/tests/test_evaluator.rs#L109 

it's changeable, I'll file a ticket to create another `version` parameter with the same value, so we can end up depricating the `app_version` for consistency with desktop